spec/requests/atmosphere/api/port_mapping_properties_spec.rb

```diff
@@ -146,7 +146,7 @@
 
     context 'when unauthenticated' do
       it 'returns 401 Unauthorized error' do
-        post api("/port_mapping_properties")
+        post api("/port_mapping_properties"), new_request
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.